internal/pkg/template/templates/environment/manifest.yml

```diff
@@ -64,6 +64,10 @@ http:
     {{- if $privateHTTP.InternalALBSubnets}}
     subnets: {{$privateHTTP.InternalALBSubnets}}
     {{- end}}
+    {{- if and $privateHTTP.Ingress $privateHTTP.Ingress.VPCIngress}}
+    ingress:
+      from_vpc: {{ $privateHTTP.Ingress.VPCIngress }}
```

Comment:
```suggestion
      from_vpc: true
```

Reply:
`{{ $privateHTTP.Ingress.VPCIngress }}` is a pointer so it check for if it exist in `if` condition and if it set then the same value gets assigned to `from_vpc` field. 
The reason behind keeping `from_vpc` as pointer because we are accepting this field from manifest where it is easier to validate the field if it pointer in order to figure out if it set or not through its `nil` value. In this case we are accepting this value from `--internal-alb-allow-vpc-ingress` as well which is not a pointer and its value is set to false when it is not being used. But if we make `from_vpc: true` in the manifest.yml then whenever `--internal-alb-allow-vpc-ingress`  is even false then also its value will be set to `$privateHTTP.Ingress.VPCIngress` pointer which is not nil anymore and it will set `from_vpc: true` which is wrong. 
Does that make sense?